### ISSUE TITLE
Image installer with customized network configuration

### ISSUE THREAD

**Issue by renne** [jens-maus/RaspberryMatic#308]
I suggest to customize the [OSMC installer](https://github.com/osmc/osmc/tree/master/installer) for RaspberryMatic. This allows the user to start the installer on his host device (MacOS, Linux, Windows), select the device type (e.g. RPi3), WiFi configuration, backup-file and mass-storage device. After that the installer writes the customized image to the mass-storage device (EMMC, SD-card, USB-stick).

**Comment by jens-maus** [comment]
I totally agree that we could do way better and providing a dedicated installer would be great! However, if this would be either a modified OSMC installer, a modified [etcher](https://etcher.io/) interface or a completely own development I actually don't know. Fact it, that it would be great to get some help here from the community as my time is way too limited to start another project for a RaspberryMatic installer. So if you or someone else could share some time and work on something like that, this would be great and highly appreciated!
